GPU/GPUTracking/Refit/GPUTrackingRefit.cxx

```diff
@@ -376,6 +380,8 @@ GPUd() int GPUTrackingRefit::RefitTrack(T& trkX, bool outward, bool resetCov)
     trk.NormalizeAlpha(alpha);
     prop.SetAlpha(alpha);
   } else if constexpr (std::is_same<S, TrackParCov>::value) {
+    static constexpr float kDeg2Rad = M_PI / 180.f;
+    static constexpr float kSectAngle = 2 * M_PI / 18.f;
     if (mPparam->rec.TrackReferenceX <= 500) {
       if (prop->PropagateToXBxByBz(trk, mPparam->rec.TrackReferenceX)) {
```

Comment:
Still on mat.correction: on the CPU this will use by default ``MatCorrType matCorr = MatCorrType::USEMatCorrTGeo``, i.e. slow TGeo query (although, in case of TPC volume it should be faster than average TGeo query).
To avoid this, I can change the default ``matCorr`` to ``USEMatCorrLUT``, with fall-back to using TGeo in https://github.com/AliceO2Group/AliceO2/blob/4c65d478ee4be49535ef34a00c2f72579e31a303/Detectors/Base/src/Propagator.cxx#L516 in case the mMatLUT was not set.

Reply:
@shahor02 : I am quite sure I was not using TGeo but the matLut, because my benchmark is not even aware of TGeo. I have added the forced switch here: https://github.com/AliceO2Group/AliceO2/blob/22cdc82f69322f0710d549944740667d0d3130dd/Detectors/Base/src/Propagator.cxx#L521
So for GPU and for my standalone tool it will always use the matLut.
Perhaps I should throw an exception if the setting points to TGeo, or do you think it is OK as is?
In general, I think it makes sense to use matLut by default.

Reply:
OK, if for the benchmark you have used ```GPUCA_...``` also for CPU / TrackParCov case, then you have used matLUT. 
I will change default to USEMatCorrLUT and add a fall-back to TGeo in getMatBudget (respecting your ifdef).